### PR TITLE
feat: create a splash screen

### LINKED ISSUE #2: feat: create a splash screen
The app need to have a splash screen so when the user opens the app it will show the logo.

Image

### PR DESCRIPTION
**Pull request by Gomez-Enzo**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -39,4 +40,5 @@ flutter:
   generate: true
 
   assets:
-    - assets/icons/
\ No newline at end of file
+    - assets/icons/
+    - assets/logo.png
```
Comment (NazarenoCavazzon) [inline pubspec.yaml]: create a new folder called images and put into it the logo.

Then change this line to `- assets/images/`

**Review by NazarenoCavazzon — CHANGES_REQUESTED** [review]

File: pubspec.yaml [hunk 2]
```diff
@@ -19,6 +19,7 @@ dependencies:
   flutter_bloc: ^8.0.1
   flutter_localizations:
     sdk: flutter
+  flutter_native_splash: ^2.2.9
```
Comment (tomassasovsky) [inline pubspec.yaml]: i think this should be a dev dependency, no?

**Review by tomassasovsky — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 2 above)
Comment (NazarenoCavazzon) [reply pubspec.yaml]: Yep, @Gomez-Enzo move this dependency to the dev ones

**Review by NazarenoCavazzon — APPROVED** [review]
LGTM

**Review by tomassasovsky — APPROVED** [review]
LGTM! 
tomi approves

**Pull request merged**